oap-server/server-receiver-plugin/zipkin-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/zipkin/analysis/transform/SegmentBuilder.java

```diff
@@ -145,7 +145,8 @@ private void scanSpansFromRoot(SpanObjectV2.Builder parentSegmentSpan, Span pare
         if (!isSegmentRoot && parentSegmentSpan != null) {
             spanBuilder.setParentSpanId(parentSegmentSpan.getSpanId());
         }
-        Span.Kind kind = span.kind();
+        //Add default Span.kind to SERVER in case span.kind is null
+        Span.Kind kind = span.kind() == null ? Span.Kind.SERVER : span.kind();
```

Comment:
Why set server as default?

Reply:
that doesn't make sense to me.

Reply:
span.kind() got null value. This cause null point exception.
If skip  span.kind is null value, no zipkin trace data will display. So we just force it to SERVER to continue handle zipkin data process.

Reply:
It triggers NPE, yes. But the default value is wrong. If no kind, I would say it is just a local span.

Reply:
yep exactly. local span is lack of kind. kind is not a required field. all fields are not required except trace and span ID

Reply:
@credit2go-admin Could you change the variable name too?